src/CodeGen/ODataT4CodeGenerator.tt

```diff
@@ -27,7 +27,9 @@ public static class Configuration
 
 	// This flag indicates whether to ignore unexpected elements and attributes in the metadata document and generate
 	// the client code if any. The value must be set to true or false.
-	public const bool IgnoreUnexpectedElementsAndAttributes = true;
+	public const bool IgnoreUnexpectedElementsAndAttributes = true;	
+	// Here you can set custom http headers like this: [{'headername': 'headervalue'}, {'headername': 'headervalue' }]

```

Comment:
It would be nice if you can add to the comment for the configuration about when this will be consumed by T4.

Reply:
I think this is done.